utils/merge-rulesets.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 
-# Merge all the .xml rulesets into a single "default.rulesets" file -- this
-# prevents inodes from wasting disk space, but more importantly, works around
-# the fact that zip does not perform well on a pile of small files.
+# Merge all the .xml rulesets into a single "default.rulesets" file -- this prevents inodes from wasting disk space, but more importantly,
+# works around the fact that zip does not perform well on a pile of small files.
```

Comment:
Why this change?

Reply:
There's no need to limit line length to 80 chars.

Reply:
All modern editors allow the user to choose the word wrap.

Reply:
Yet most styles still prefer it.  Take PEP 8, which instructs wrapping at 79 characters.  Wrapping a comment line can be quite ugly in some editors.  Let's keep the wrapping, it's orthogonal to the separate change in this PR.

Reply:
@Hainish Done.